tests/Unit/Printer/FinalPrinterTest.php

```diff
@@ -55,6 +60,51 @@ public function testOnEngineEndPrintsTheRightCountSummary(): void
         $this->assertStringContainsString('Executed: 5 test classes (1 retried), 44 tests', $output->getOutput());
     }
 
+    public function testOnEngineEndPrintsTheRightChunkedCountSummary(): void
+    {
+        ClockMock::register(Stopwatch::class);
+        ClockMock::register(__CLASS__);
+        $output = new UnformattedOutputStub();
+        $chunkSize = $this->prophesize(ChunkSize::class);
+        $chunkSize->isChunked()
+            ->shouldBeCalled()
+            ->willReturn(true);
+
+        $testResultContainer = $this->prophesize(TestResultContainer::class);
+        $testResultContainer->countTestResults()
+            ->willReturn(3);
+        $testResultContainer->getTestResults()
+            ->willReturn(array_fill(0, 3, $this->mockTestResult()));
+        $testResultContainer->getTestResultFormat()
+            ->willReturn($this->mockTestFormat());
+        $testResultContainer->getFileNames()
+            ->willReturn(['Test.php']);
+
+        $testResultList = $this->prophesize(TestResultList::class);
+        $testResultList->getTestResultContainers()
+            ->willReturn(array_fill(0, 15, $testResultContainer->reveal()));
+
+        $printer = new FinalPrinter($testResultList->reveal(), $output, $chunkSize->reveal());
+
+        ClockMock::withClockMock(true);
+
+        $printer->onEngineStart();
+        $printer->onProcessTerminated();
+        $printer->onProcessTerminated();
+        $printer->onProcessTerminated();
+        $printer->onProcessTerminated();
+        $printer->onProcessTerminated();
+        $printer->onProcessToBeRetried();
+        $printer->onProcessTerminated();
+        usleep(60499999);
```

Comment:
Why this sleep? Aren't we using the `ClockMock`?

Reply:
Here I did copy/paste from `testOnEngineEndPrintsTheRightCountSummary()` - I think usleep here is using mocked version from ClockMock to mock execution time of 1 minute ('Execution time -- 00:01:00' in expectations). 

But I think this test `testOnEngineEndPrintsTheRightChunkedCountSummary()` could be simplified to just test the output part and skip ClockMock'ing etc...

Reply:
Updated changelog, but I didn't specify release date - please update this.